draft-ietf-teep-architecture.md

```diff
@@ -251,9 +251,10 @@ The following terms are used:
     that one TA cannot read, modify or delete the data and code of another
     TA.
 
-  - Untrusted Application: An application running in a Rich Execution
-    Environment.
-
+  - Untrusted Application: An application running in a REE. An Untrusted Application 
```

Comment:
```suggestion
  - Untrusted Application: An application running in an REE. An Untrusted Application 
```
"an REE" occurs 3 times in the doc, and "a REE" once (besides here).  We should be consistent.